manual/manual/refman/lex.etex

```diff
@@ -30,12 +41,8 @@ letter: "A" \ldots "Z" || "a" \ldots "z"
 Identifiers are sequences of letters, digits, "_" (the underscore
 character), and "'" (the single quote), starting with a
 letter or an underscore.
-Letters contain at least the 52 lowercase and uppercase
-letters from the ASCII set. The current implementation
-also recognizes as letters some characters from the ISO
-8859-1 set (characters 192--214 and 216--222 as uppercase letters;
-characters 223--246 and 248--255 as lowercase letters). This
-feature is deprecated and should be avoided for future compatibility.
+(Currently, letters consist of the 52 lowercase and uppercase
+letters from the ASCII character set.)
```

Comment:
These parentheses does not seem warranted. I would also remove `Currently` which makes the sentence unnecessarily controversial.

Reply:
Easily fixed.

Reply:
Done.